data/human/free worlds 0 prologue.txt

```diff
@@ -1288,18 +1288,25 @@ mission "FW Katya 1"
 			
 			label karate
 			`	You dodge sideways and spin quickly. As you do so you hear a loud click, but the gun does not fire. With one hand you knock aside the gun. With the other you land a karate chop which, if you had studied karate, would've shattered her collarbone and left her reeling in pain. But since you have not studied karate, instead your blow bounces harmlessly off her surprisingly muscular shoulder.`
-			`	Your assailant turns out to be wearing a Free Worlds uniform and a disappointed expression. "Well aren't you a dumb reckless bastard!" she says, "I'd hoped for something better from you. First of all, if this gun were loaded, your brains would be splattered all over the sidewalk. And second, you fight like a girl. I have no use for hotheads like you." She stalks off, leaving you very confused. It would appear that you've just failed some sort of test.`
+			`	Your assailant turns out to be wearing a Free Worlds uniform and a very disappointed expression. "Well, aren't you a dumb, reckless bastard." she says, "I'd hoped for something better from you. Were this gun loaded, your brains would already be splattered all over the sidewalk. And you hit like a paper tiger that's been soaking for days. I have no use for hotheads like you." She makes to stalk off, leaving you very confused. It would appear that you've just failed some sort of test.`
```

Comment:
```suggestion
			`	Your assailant turns out to be wearing a Free Worlds uniform and a very disappointed expression. "Well, aren't you a reckless fool." she says, "I'd hoped for something better from you. First of all, if this this gun were loaded, your brains would already be splattered all over the sidewalk. And second, you hit like a paper tiger that's been soaking for days. I have no use for hotheads like you." She stalks off, leaving you very confused. It would appear that you've just failed some sort of test.`
```
The irony of picking at this example of a sexist phrase despite what it can mean to characterization (including when contrasted against later representation of feminine warrior, meaning this is about stereotypes of "girl" as a specific gender representation [possibly specific to humanity as a culture/species], not an assumption from the character of the fighting ability of all females) but not addressing the ableism or denigration of heritage through the institution of marriage in the very same paragraph.

Suggesting to revert unrelated and unnecessary style adjustments to it as well, especially in the dialogue itself. (I do agree that the ! is improperly matched with "said," though, but would prefer a better speech verb to restore the ! instead, as punctuation speaks to intent more than poor word choice.)